app/views/static_pages/home.html.erb

```diff
@@ -1,4 +1,3 @@
-<% provide :title, "Home" %>
```

Comment:
sao lại xóa dòng này

Reply:
Vì em không muốn home page in ra title "Home | Rails Tutorial" mà chỉ muốn in ra "Rail Tutorial" thôi ạ